flow/project.py

```diff
@@ -689,9 +671,21 @@ def _determine_entrypoint(self, entrypoint, directives, job):
 
     def _resolve_directives(self, name, defaults):
```

Comment:
When this also evaluates the values, we should also change the `get_directives` function in `_get_submission_directives` to not check for callable since it will have already been evaluated.